api/src/Entity/Repairer.php

```diff
@@ -115,10 +115,12 @@ class Repairer
     #[Groups([self::REPAIRER_READ, self::REPAIRER_WRITE, self::REPAIRER_COLLECTION_READ])]
     public ?User $owner = null;
 
-    #[ORM\ManyToOne]
-    #[ORM\JoinColumn(nullable: true)]
+    #[ORM\JoinTable(name: 'repairer_repairer_type')]
```

Comment:
Do we need the jointTable ? Doctrine could not detect it automatically ? Do not remember if we add this attribute for others manyToMany in the project